include/geometrycentral/surface/manifold_surface_mesh.h

```diff
@@ -65,12 +64,11 @@ class ManifoldSurfaceMesh : public SurfaceMesh {
   // Collapse an edge. Returns the vertex adjacent to that edge which still exists. Returns Vertex() if not
   // collapsible. Assumes triangular simplicial complex as input (at least in neighborhood of collapse).
   Vertex collapseEdge(Edge e);
+  
+  bool collapseEdgeTriangular(Edge e, Vertex &v);
```

Comment:
Could this be changed to have a signature like:
```
Vertex collapseEdgeTriangular(Edge e);
```
and return the null `Vertex()` when impossible? This call style is used in other mesh mutation operations so it would be nice to be consistent.

Also, the plain old `collapseEdge()` right above also assumes triangular... should both still exist? Is that old `collapseEdge()` obsolete now?

Reply:
Done. Also I think, if the old edge collapse also assumes triangular, then it's obsoleted by this one. Assuming it isn't referenced anywhere else in the code, it should be safe to delete.